components/humantask/org.wso2.carbon.humantask/src/main/java/org/wso2/carbon/humantask/core/store/HumanTaskStore.java

```diff
@@ -692,11 +692,14 @@ private AxisService createAxisService(WSDL11ToAxisServiceBuilder serviceBuilder,
             operation.setMessageReceiver(msgReceiver);
             getTenantAxisConfig().getPhasesInfo().setOperationPhases(operation);
         }
-        List<String> transports = new ArrayList<String>();
-        transports.add(Constants.TRANSPORT_HTTPS);
-        transports.add(Constants.TRANSPORT_HTTP);
-        transports.add(Constants.TRANSPORT_LOCAL);
-        axisService.setExposedTransports(transports);
+
+        Set<String> exposedTransports = getTenantAxisConfig().getTransportsIn().keySet();
+        /**
+         * Add the transports to axis2 service by reading from the tenant transport config
```

Comment:
better replacing with Single-Line Comment since this is not function description and it's a short comment